### DISCUSSION TITLE
[bug] Recreate Group from `toObject()` method

### DISCUSSION THREAD

**Discussion by DocAmz** [fabricjs/fabric.js#10012] [Q&A] [answered]
Hi fabricJS community ! 

I just update fabricJS in my project : form fabric.JS 6.0.0-beta15 ➡️  fabric.JS 6.0.2
Congrats to the team for the release 🥳  !

## Description

For different features (Undo/redo, multiple page,... ) i'm using `toObject` method to store the objects until they need to be instantiated back. This was working perfectly with the version 6.0.0-beta15

When i try to recreate `Group` from stored objects with `toObject` method  this error is thrown 

```javascript
TypeError: this.layoutManager.performLayout is not a function
```

## Step to reproduce 
Use the **Store & retrive** button

[CodeSandbox](https://codesandbox.io/p/sandbox/str-storeandretrive-group-np9s69)

## Additional Context
- **Fabric Version:** "fabric": "6.0.2"
- **Typescript Version:** "typescript": "5.2.2"

I don't have identified a potential fix. 
Can someone assist with resolving this issue?

Thanks !

**Comment by asturur** [comment, marked as answer]
I see you are restoring the group in your own way rather than using the fromObject function.
If you do that you have either to pass a valid layout manager or not passing it at all.
By passing the options in the group constructor you are passing as layout manager its own serialized version of it, erroring.

**Reply by DocAmz** [reply]
I understand the error here.
I will fix this by removing the serialized layout manager for now, and handle this later.

Thanks for your answer 👍
